_docs/layout/navigation.md

```diff
@@ -135,65 +149,70 @@ for highly compliant Helix applications is the [left navigation](#side-nav).
 {% endcolumn %}
 </div>
 
-### Top Navigation Spacing
+### Top navigation spacing
 
 <div class="row">
 {% column left:"col-md-4" %}
--   Horizontal spacing includes 20px left margin
 
--   Each menu item button has 20px left and right padding
+-   Horizontal spacing includes 20px of left margin.
+
+-   Each menu item button has 20px of left and right padding.
 
-    {% include note.html content="Dropdown padding left begins at the left
-    border of the parent primary navigational item and padding right begins
-    after the longest textual element in the dropdown element." %}
+    {% include note.html content="Drop-down element left padding
```

Comment:
this could use laymans terms
e.g. 
The left padding of a dropdown begins at the left border of the parent primary navigational item and padding right begins after the longest textual element in the dropdown element.